pkg/openstack/telemetry.go

```diff
@@ -27,6 +27,19 @@ const (
 	telemetryName           = "telemetry"
 )
 
+func ensureKsmImageDefault(version *corev1beta1.OpenStackVersion) *string {
+	// NOTE(mmagr): for case upgrade case where KSM image is not part of OpenStackVersion
+	// yet we temporarily need to ensure image default is passed
+	var ksmImageTempDefault = "registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.10.0"
```

Comment:
I think we'll want to avoid using hard coded images as it would need to be set in RELATED_IMAGES as well to ensure it gets downloaded for offline/airgapped.